Dockerfile.ubuntu_builder

```diff
@@ -0,0 +1,46 @@
+# syntax=docker/dockerfile:1
```

Comment:
What's the point of keeping (and maintaining) this one? if you have manylinux wheels you can use them on all the ubunti

Reply:
Well you can ignore it for now but it might be a more robust alternative for Ubuntu users and it's a little simpler so might also be a useful reference for when the build system changes.